### ISSUE TITLE
UnicodeDecodeError when trying to decode compiler output on Windows (was: Problem with importer.create_problem() function)

### ISSUE THREAD

**Issue by feddersmichael** [AMICI-dev/AMICI#1307]
I executed the example doc [sampling_diagnostics](https://github.com/ICB-DCM/pyPESTO/blob/master/doc/example/sampling_diagnostics.ipynb) from PyPesto and the first code cell did not work as intended.
I got an error while executing the last code line in the first code cell `problem = importer.create_problem()`
The exact error code was 

UnicodeDecodeError: 'utf-8' codec can't decode byte 0x94 in position 66692: invalid start byte

I added the complete output in a seperate PDF-file.
[Amici-Issue.pdf](https://github.com/AMICI-dev/AMICI/files/5395395/Amicic-Issue.pdf)

I am using AMICI Version 0.11.7 and PyPesto Version 0.2.2 on an Anaconda Python Interpreter on Windows.
Thanks for any support!

**Comment by dweindl** [comment]
Hi Michael, can you please check whether it works with https://github.com/AMICI-dev/AMICI/pull/1308/commits/c12fca87853dbc014c5308597c1dbec34b87c947?

**Comment by feddersmichael** [comment]
I changes the file ode_export accordingly, but unfortenately now I get the following error.
[Amici-issue_2.pdf](https://github.com/AMICI-dev/AMICI/files/5397424/Amici-issue_2.pdf)

**Comment by dweindl** [comment]
> I changes the file ode_export accordingly, but unfortenately now I get the following error.
> [Amici-issue_2.pdf](https://github.com/AMICI-dev/AMICI/files/5397424/Amici-issue_2.pdf)

```
  File "C:\Anaconda3\envs\Bachelor-Thesis\lib\site-packages\amici\ode_export.py", line 11
    ximport sympy as sp
            ^SyntaxError: invalid syntax
``` 

I think you accidentally changed too much. There shouldn't be an `x`.

**Comment by feddersmichael** [comment]
I am sorry, I must have misclicked myself there.
After changing this i still get the same error message. I also added a screenshot from teh changed code, maybe I made a mistake there as well.
[Amici-Issue.pdf](https://github.com/AMICI-dev/AMICI/files/5397456/Amici-Issue.pdf)
![code](https://user-images.githubusercontent.com/56444515/96367028-a138b680-114b-11eb-8271-7efe36beb8e0.png)

**Comment by dweindl** [comment]
The changed lines are fine, but don't do the job. What about 8bf05b92e912672ef4d5fb2b073052a5a648bb45 ?

**Comment by dweindl** [comment]
> The changed lines are fine, but don't do the job. What about [8bf05b9](https://github.com/AMICI-dev/AMICI/commit/8bf05b92e912672ef4d5fb2b073052a5a648bb45) ?

I mean 199ecb50f86e0e7b9ba62d828c539a94a5a5d571

**Comment by feddersmichael** [comment]
I still get the same error.
[Amici-Issue.pdf](https://github.com/AMICI-dev/AMICI/files/5397541/Amici-Issue.pdf)

**Comment by dweindl** [comment]
> (It was intended to only apply 199ecb5 and not 8bf05b9 right?)

Yes.

Okay, sorry. Maybe somebody with access to a windows machine can look into that... 


For a workaround it should be fine to replace those two lines by `print(result.stdout)`.

**Comment by thomassligon** [comment]
Just a little remark, even if it doesn't help any. I tried this on my PC, Windows with Visual Studio with the corresponding python 3.7 and Visual C++, amici 0.11.7, petab 0.1.10 and pypesto 0.2.2. Everything worked fine without any changes. I didn't find the character that python is complaining about, because it doesn't give us the file name, only the displacement (66692). On the other hand, I had to change my SBML model because python doesn't accept Greek letters.

**Comment by feddersmichael** [comment]
Okay so I shotdown the kernel again and restarted the code cell and this time i got a different error - not sure if I should open a new issue or this one is correlating with the latter fix.

[Amici-Issue.pdf](https://github.com/AMICI-dev/AMICI/files/5397583/Amici-Issue.pdf)

I got the same  error also while trying to start the code cells [here](http://localhost:8888/notebooks/doc/example/conversion_reaction.ipynb), [here](http://localhost:8888/notebooks/doc/example/sampler_study.ipynb) and [here](http://localhost:8888/notebooks/doc/example/petab_import.ipynb).

**Comment by dweindl** [comment]
Makes sense. Oops. The first line needs to be `print(e.output)`, the second line is fine.

**Comment by feddersmichael** [comment]
When you refer to the fist one you mean the one directly after `except subprocess.CalledProcessError as e:`?

**Comment by dweindl** [comment]
yes

**Comment by feddersmichael** [comment]
Okay, now I get this one:
[Amici-Issue.pdf](https://github.com/AMICI-dev/AMICI/files/5397589/Amici-Issue.pdf)

**Comment by feddersmichael** [comment]
This is the complete output:
[Amici-Issue.pdf](https://github.com/AMICI-dev/AMICI/files/5397597/Amici-Issue.pdf)

**Comment by FFroehlich** [comment]
closing in favor of #1309

**Comment by dweindl** [comment]
> closing in favor of #1309

The encoding issue here is unfortunately not fixed yet (independent of #1309). Reopening.
